lib/composer.js

```diff
@@ -194,6 +194,10 @@ export default class Composer extends Disposable {
 
     await Promise.all(jobs)
 
+    if (this.shouldCleanAfterBuild()) {
+      this.clean()
```

Comment:
Thanks for pointing this out. Excuse me for the mistakes I do as I try to grasp Javascript on the way 🙈